docs/user_guide/theme-elements.md

```diff
@@ -118,6 +118,17 @@ I was made with the `{admonition}` directive and a `sidebar` class.
 I was made with the `{sidebar}` directive.
 ```
 
+## Footnotes
+
+Here's one footnote[^1] and another footnote [^2] and a named footenote[^named], symbol [^*].
+
+[^1]: Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar.
+[^2]: Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar.
+[^named]: Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar.
+[^*]: Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar. Foo bar foo bar.
+
+## Version changes
+
```

Comment:
This doesn't look correct? It is a whole section inserted into the middle of another one (Admonition sidebars). It also restores the `Version changes` heading that was removed in https://github.com/pydata/pydata-sphinx-theme/pull/836/files#diff-0607a043e4091f4e22ccc62684762b4a6b00d03e638ff096277e7f4dde9cba77L97, but none of its section content.